src/app_charts/base/cloud/app-management.yaml

```diff
@@ -85,6 +85,33 @@ metadata:
   annotations:
     cert-manager.io/inject-ca-from: {{ .Release.Namespace }}/app-rollout-controller
 webhooks:
+- name: apps.apps.cloudrobotics.com
+  admissionReviewVersions: ["v1"]
+  failurePolicy: Fail
+  clientConfig:
+    service:
+      namespace: {{ .Release.Namespace }}
+      name: app-rollout-controller
+      path: /app/validate
+  rules:
+  - apiGroups:
+    - apps.cloudrobotics.com
+    apiVersions:
+    - v1alpha1
+    operations:
+    - CREATE
+    - UPDATE
+    resources:
+    - apps
+  sideEffects: None
+---
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingWebhookConfiguration
+metadata:
+  name: app-rollout-controller
```

Comment:
I don't think you can have this twice, but the content of webhooks is a list, so maybe you can merge this?

Reply:
Thanks for pointing that out, even though it built, it was getting overwritten. And there was a fixup needed for the hook implementation itself. I merged them now
```
NAME                     WEBHOOKS   AGE
app-rollout-controller   2          7d5h
```